test/unittest/blas2/blas2_gemv_test.cpp

```diff
@@ -26,7 +26,7 @@
 #include "blas_test.hpp"
 
 template <typename T>
-using combination_t = std::tuple<int, int, bool, T, T, int, int, int>;
+using combination_t = std::tuple<int, int, T, T, bool, int, int, int>;
```

Comment:
Is there a reason argument order was changed?

Reply:
yes because the order was wrong